src/healthkit.cs

```diff
@@ -3224,16 +3571,19 @@ interface HKWorkoutRouteBuilder {
 		void AddMetadata (HKMetadata metadata, HKWorkoutRouteBuilderAddMetadataHandler completion);
 	}
 
-	[Watch (4,0), iOS (11,0)]
+	[Watch (4,0), iOS (11,0), Mac (13,0)]
 	[BaseType (typeof(HKQuery))]
 	interface HKWorkoutRouteQuery {
 		[Export ("initWithRoute:dataHandler:")]
-		[DesignatedInitializer]
```

Comment:
Are there no designated initializers anymore in this class?

Reply:
@rolfbjarne hmm, I do not see anything in the headers that suggests there is a new one